frontend/document/koptinterface.lua

```diff
@@ -1433,7 +1433,7 @@ end
 local function get_pattern_list(pattern, case_insensitive)
     -- pattern list of single words
     local plist = {}
-    -- (as in util.splitToWords(), but only splitting on spaces, keeping punctuations)
```

Comment:
Possibly not a typo, in which case the author might prefer something like `punctuation marks`.

Reply:
Oh, that's me :)
I prefer "punctuation marks" if "punctuations" is wrong standalone.